File: POO/Java/UberVan.java
package POO.Java;

import java.util.Map;
import java.util.ArrayList;

public class UberVan extends Car {
    
    private Map<String, Map<String,Integer>> typeCarAccepted;
    private ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver,
                    Map<String, Map<String,Integer>> typeCarAccepted,
                    ArrayList<String> seatsMaterial) {
        super(license,driver,1);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: POO/PHP/UberVan.php
<?php
    require_once('Car.php');

    class UberVan extends Car{
        public $typeCarAccepted;
        public $seatsMaterial;

        public function __construct($license,$driver),$typeCarAccepted,$seatsMaterial){
            parent::__construct($license,$driver),6;
            $this->$typeCarAccepted=$typeCarAccepted;
            $this->$seatsMaterial=$seatsMaterial;
        }
    }
?>